### PR TITLE
feat: Change cg scale

### LINKED ISSUE #780: Change admonition for CG sclae in cockpit
See https://github.com/flybywiresim/a32nx/pull/7931

### PR DESCRIPTION
**Pull request by Luz5020**
Fixes #780
## Summary

Changes docs for [this PR](https://github.com/flybywiresim/a32nx/pull/7931), also see attached Issue #780 for additional information
### Location

https://docs.flybywiresim.com/pilots-corner/beginner-guide/engine-start-taxi/
https://docs.flybywiresim.com/pilots-corner/a32nx-briefing/flight-deck/pedestal/thrust-pitch-trim/

Discord username (if different from GitHub):

### CODE REVIEW

File: docs/pilots-corner/beginner-guide/engine-start-taxi.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ Perform the AFTER START checklist.
 
         The FlyByWire checklist's trim section at the bottom indicates we would need to set a nose up trim of about 0.2.
 
-    !!! warning "The CG scale available in the Asobo's model of the flight deck not entirely accurate for the A320neo."
+    !!! warning "The CG scale available in the Asobo's model of the flight deck not entirely accurate for the A320neo. This is already fixed in the [Development Version](https://docs.flybywiresim.com/fbw-a32nx/fbw-versions/#development-version-recommended) of the Plane."
```
Comment (Sleinmaster) [inline docs/pilots-corner/beginner-guide/engine-start-taxi.md]:
```suggestion
    !!! warning "The CG scale available in the Asobo's model of the flight deck not entirely accurate for the A320neo. This is corrected in the [Development Version](https://docs.flybywiresim.com/fbw-a32nx/fbw-versions/#development-version-recommended) of the Plane."
```

**Comment by Valastiri** [comment]
https://docs-git-fork-luz5020-change-cg-scale-flybywire.vercel.app/pilots-corner/a32nx-briefing/flight-deck/pedestal/thrust-pitch-trim/

can you make the change on this page as well? -- and the image is slightly not straight. I could fix that on photoshop if you need assistance just let me know here or on Discord.

File: docs/pilots-corner/beginner-guide/engine-start-taxi.md [hunk 2]
```diff
@@ -209,7 +209,7 @@ Perform the AFTER START checklist.
 
         The FlyByWire checklist's trim section at the bottom indicates we would need to set a nose up trim of about 0.2.
 
-    !!! warning "The CG scale available in the Asobo's model of the flight deck not entirely accurate for the A320neo."
+    !!! warning "The CG scale available in the Asobo's model of the flight deck not entirely accurate for the A320neo. This is corrected in the [Development Version](https://docs.flybywiresim.com/fbw-a32nx/fbw-versions/#development-version-recommended) of the Plane."
```
Comment (Valastiri) [inline docs/pilots-corner/beginner-guide/engine-start-taxi.md]:
```suggestion
    !!! warning "The CG scale available in the Asobo model of the flight deck is not entirely accurate for the A320neo. This is corrected in the [Development Version](https://docs.flybywiresim.com/fbw-a32nx/fbw-versions/#development-version-recommended) of the Plane."
```

Comment (Valastiri) [reply docs/pilots-corner/beginner-guide/engine-start-taxi.md]: Minor suggestion

**Comment by Luz5020** [comment]
> https://docs-git-fork-luz5020-change-cg-scale-flybywire.vercel.app/pilots-corner/a32nx-briefing/flight-deck/pedestal/thrust-pitch-trim/
> 
> can you make the change on this page as well? -- and the image is slightly not straight. I could fix that on photoshop if you need assistance just let me know here or on Discord.

Is it not? I got it as straight as possible, and was focussing on the lighting, if you can fix it go ahead, it beats retaking it

**Comment by Valastiri** [comment]
thanks for the fix!

**Pull request merged**